src/vcpkg/input.cpp

```diff
@@ -35,8 +35,7 @@ namespace vcpkg
         if (!paths.is_valid_triplet(t))
         {
             msg::println_error(msgInvalidTriplet, msg::triplet = t);
-            LockGuardPtr<Metrics>(g_metrics)->track_string_property(StringMetric::Error,
-                                                                    "invalid triplet: " + t.to_string());
+            get_global_metrics_collector().track_string(StringMetric::Error, "invalid triplet: " + t.to_string());
```

Comment:
Hmm, we must suppress this datapoint and replace it with some new metric that ensures no PII can make it through.

Reply:
In that case I think we should just remove it completely and add specific define metrics for the specific errors we know about rather than having a catch-all string metric for errors?